constant/index.js

```diff
@@ -1,6 +1,6 @@
 export const ONE_DATE_IN_MS = 86400000;
 
-export const SUPPORTED_IMAGE_TYPE = new Set(['jpg', 'png']);
+export const SUPPORTED_IMAGE_TYPE = new Set(['jpeg', 'png']);
```

Comment:
How about gif...?

Reply:
Kiu has replied that gif is not supported.